notebooks/official/model_evaluation/model_based_llm_evaluation/autosxs_check_alignment_against_human_preference_data.ipynb

```diff
@@ -0,0 +1,853 @@
+{
```

Comment:
<p>Use&nbsp;<a href="https://developers.google.com/style/capitalization#capitalization-in-titles-and-headings" target="_blank">sentence case to capitalize titles and headings</a>: Generate evaluation dataset for AutoSxS human alignment checking </p><p> </p><p>####Note: Add a space after # for markdown to work</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/2659/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='33'/>